package.json

```diff
@@ -5,19 +5,17 @@
     "export": "next export",
     "deploy": "npm run build && npm run export",
```

Comment:
Do we use the `deploy script`? If not, we should dump it. If we do use it, we'll need to update it to use yarn if we move forward with that change.